packages/xo-server/src/api/pool.mjs

```diff
@@ -334,6 +334,22 @@ getLicenseState.resolve = {
 
 // -------------------------------------------------------------------
 
+export async function getGuestSecureBootReadiness({ pool }) {
+  return this.getXapi(pool).call('pool.get_guest_secureboot_readiness', pool._xapiRef)
```

Comment:
This method is not available in older XAPIs, and will generate `MESSAGE_METHOD_UNKNOWN` errors.

This will create a lot of failed XO tasks (due to the subscription), how do you want to handle this?